homeassistant/components/conversation/__init__.py

```diff
@@ -389,7 +389,8 @@ def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the conversation agents."""
         self.hass = hass
         self._agents: dict[str, AbstractConversationAgent] = {}
-        self._default_agent_init_lock = asyncio.Lock()
+        self._builtin_agent_init_lock = asyncio.Lock()
+        async_setup_default_agent(hass)
```

Comment:
Do you consider this a side effect and should we then have an explicit setup method for the instance?

Reply:
Yes, you're right.
Fixed! 👍